perf/load/templates/istio-ingress.gen.yaml

```diff
@@ -26,10 +26,16 @@ spec:
   - route:
     - destination:
         host: {{ .Values.serviceNamePrefix }}0
-        subset: all
+        subset: v1
         port:
           number: 8080
-      weight: 100
+      weight: 30
+    - destination:
```

Comment:
what's all this for? rest looks good

Reply:
to have at least one traffic split in the default config.